content/blog/using-polymorphic-json-adapter-on-android.md

```diff
@@ -123,15 +122,16 @@ Now, if the app receives a request where the `occupation` is Doctor or Engineer,
    }
    ```
 
-   will generate an instance of `Doctor` class and similarly if the occupation was Engineer it would have generated an instance of `Engineer` class.
-3.   Now, the last task is just adding your PolymorphicJsonAdapterFactory to the Moshi builder and then using that Moshi instance wherever we want (e.g. Retrofit)
+   will generate an instance of the `Doctor` class.
+
+3.   The final step is to add this adapter to the builder of your `Moshi` instance.
```

Comment:
Self Note: Add another point to let users know that they would still need adapters for the subclasses of `Person`. They can use codegen or reflection.